src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java

```diff
@@ -297,7 +297,8 @@ private void start() throws ReaperException, InterruptedException {
   private void updateClusterNodeList() throws ReaperException {
     Set<String> liveNodes  = ImmutableSet.copyOf(clusterFacade.getLiveNodes(cluster));
     Cluster cluster = context.storage.getCluster(clusterName);
-    if (!cluster.getSeedHosts().equals(liveNodes) && !liveNodes.isEmpty()) {
+    if (context.config.getEnableDynamicSeedList() && !cluster.getSeedHosts().equals(liveNodes)
```

Comment:
can you add a comment referencing the `ClusterResource` logic that we're keeping in sync with here…?